source/_integrations/dsmr.markdown

```diff
@@ -64,6 +64,12 @@ This integration is known to work for:
 - Ziv E0058 ESMR5
 - EasyMeter Q3D
 
+### MBUS Support
```

Comment:
```suggestion
### M-Bus support
```
That's how it is spelled on the M-Bus website: https://m-bus.com/. Please correct me if I got this wrong.